src/wp-admin/includes/update-core.php

```diff
@@ -1574,6 +1577,52 @@ function _preload_old_requests_classes_and_interfaces( $to ) {
 	}
 }
 
+/**
+ * Preloads old classes and interfaces required for updating Core.
+ *
+ * This function preloads the old code into memory before the
+ * upgrade process deletes the files. These files are loaded into
+ * memory via an autoloader, meaning when a class or interface is needed
+ * WordPress Core could attempt to access code. If the file is not there,
+ * a fatal error could occur. If the file was replaced, the new code is not
+ * compatible with the old, resulting in a fatal error.
+ * Preloading ensures the code is in memory before the code is updated.
+ *
+ * @since 6.6.0
+ */
+function _preload_core_classes_and_interfaces() {
+	// The classes to be preloaded.
+	// These are lowercased to match the format used in the autoloader.
```

Comment:
Needs to follow multi line comment standard per coding standards https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#5-inline-comments

Are you able to do a search for `//` and `/*` throughout the PR to check these.

Reply:
Thank you @peterwilsoncc 

I went through the diff and fixed any comments-formatting issues I found 👍